src/coreclr/inc/gcinfodecoder.h

```diff
@@ -186,6 +186,7 @@ enum ICodeManagerFlags
     NoReportUntracked
                     =   0x0080, // EnumGCRefs/EnumerateLiveSlots should *not* include
                                 // any untracked slots
+    ReportFPBasedSlotsOnly = 0, // Unused by nativeaot, the gc info decoder checks that. Set to 0 to let the compiler optimize out the related code.
```

Comment:
```suggestion
    ReportFPBasedSlotsOnly
                    =   0x0200, // EnumGCRefs/EnumerateLiveSlots should only include
                                // slots that are based on the frame pointer
```
I assumed that the redundant definition is going to be somewhere under nativeaot. If it is here, I would make it match the other definition and live with the tiny bit of extra code.